campaignion_opt_in/campaignion_opt_in.module

```diff
@@ -165,3 +165,10 @@ function campaignion_opt_in_webform_submission_load(&$submissions) {
     $submission->opt_in = new Values(new Submission($node, $submission));
   }
 }
+
+/**
+ * Implements hook_webform_submission_create_alter().
+ */
+function campaignion_opt_in_webform_submissions_create_alter($submission, $node) {
```

Comment:
`submissions` in plural in the function name seems wrong...

Reply:
Where/why is this alter hook necessary?

Reply:
This way I can rely on always having `$submission->opt_in` (especially in `hook_webform_submission_insert()`).

I have fixed the typo and added tests.